application/modules/sessions/controllers/Sessions.php

```diff
@@ -22,11 +22,15 @@ public function index()
 
     public function login()
     {
+        $file = 'application/logs/loginlog.txt';
         $view_data = array(
             'login_logo' => get_setting('login_logo')
         );
 
         if ($this->input->post('btn_login')) {
+            $log_time = time();
+            $log_line = $_SERVER['REMOTE_ADDR'] . ' ';
```

Comment:
@Kovah what's the point on this line ? It miss a dot before the equal to append the IP no ? Or the line 31 is not needed as it's re-defined in this line ? ;)

Reply:
This is outdated. I moved the logging into the session model. 
See 7070f20b0f13abe9960b87f58d243a956871e904

Reply:
Indeed I just saw the merge tag. :+1: